server/apps/main/views.py

```diff
@@ -11,6 +11,7 @@
 from django.db.models import Q
 from django.urls import reverse
 from django.core.paginator import Paginator
+from django.core.exceptions import EmptyResultSet, ObjectDoesNotExist
```

Comment:
```suggestion
from django.core.exceptions import ObjectDoesNotExist
```

In the original PR doesn't need either as unused. With my suggested edit (below), need the `ObjectDoesNotExist` exception